lerobot-web-fe/src/components/Layout.tsx

```
@@ -0,0 +1,42 @@
+import { DrawerMenu } from './DrawerMenu';
+import Header from './Header';
+import styles from './Layout.module.css';
+import Teleoperate from './Teleoperate';
+import Calibration from './Calibration';
+import Policies from './Policies';
+import AITraining from './AITraining';
+import { BrandLogo } from './BrandLogo';
+import { MainScene } from './MainScene';
+import Robot from './Robot';
+
+const pages = [<Teleoperate />, <Calibration />, <Policies />, <AITraining />];
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)